next.config.js

```diff
@@ -90,12 +90,16 @@ module.exports = (phase, { defaultConfig }) => {
       remotePatterns: [
         {
           protocol: "https",
-          hostname: "crowdin-static.downloads.crowdin.com",
+          hostname: "crowdin-static.cf-downloads.crowdin.com",
         },
         {
           protocol: "https",
           hostname: "avatars.githubusercontent.com",
         },
+        {
+          protocol: "https",
+          hostname: "github.com",
+        },
```

Comment:
Hm... curious where this is needed? Small note, but keeping in mind any domain patterns we allow here would allow others to use our next server to optimize their images...

ie, anyone could use our servers to optimize any github-hosted image, using this pattern: `https://ethereum.org/_next/image/?url=<any-github-hosted-image-url>&w=<size>&q=<quality>`